R/tidiers_ml_lda.R

```diff
@@ -0,0 +1,59 @@
+#' Tidying methods for Spark ML LDA models
+#'
+#' These methods summarize the results of Spark ML models into tidy forms.
+#'
+#' @param x a Spark ML model.
+#' @param ... extra arguments (not used.)
+#' @name ml_lda_tidiers
+NULL
+
+#' @rdname ml_lda_tidiers
+#' @export
+tidy.ml_lda_model <- function(x,
+                              ...){
+
+ doc_concentration <- sparklyr::invoke(x$model$estimated_doc_concentration(),
```

Comment:
Let's put some line breaks in these functions calls, see https://style.tidyverse.org/syntax.html#long-lines.